engine/src/flutter/shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -1204,6 +1205,52 @@ public void setPaddingTopToZeroForFullscreenModeLegacy() {
     validateViewportMetricPadding(viewportMetricsCaptor, 100, 0, 100, 0);
   }
 
+  @Test
+  @TargetApi(31)
+  @Config(sdk = API_LEVELS.API_31)
```

Comment:
```suggestion
  @Config(minSdk = API_LEVELS.API_31)
```

Just to ensure it works on 31 and above :)